OpenNefia.Core/SaveGames/SaveGameSerializer.cs

```diff
@@ -399,5 +395,12 @@ public void LoadGlobalData(ISaveGameHandle save)
                 }
             }
         }
+
+        public void ResetGame()
```

Comment:
I think this should be called `ResetGameState()` instead.

Reply:
changed in 2e3ad22